README.md

```diff
@@ -624,9 +624,9 @@ Vim has a lot of nifty tricks and we try to preserve some of them:
 
 ## 📚 F.A.Q.
 
-### None of the vim `ctrl` (e.g. `ctrl+f`, `ctrl+v`) commands work
+### None of the IDE `ctrl` (e.g. `ctrl+f`, `ctrl+v`) commands work
```

Comment:
Sorry for being pedantic, but can you change ```IDE``` to ```native Visual Studio Code```

Reply:
Sure : )